.github/workflows/semantic-pr.yml

```
@@ -14,6 +14,8 @@ concurrency:
 permissions:
   pull-requests: read
 
+  permission to ci_185321: write
```

actionlint (1.7.12)

[error] unknown permission scope "permission to ci_185321". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)